test/run-test.sh

```diff
@@ -16,8 +16,9 @@ WAIT_FOR_HEARTBEAT_TIMEOUT_TEST=0
 RANDOMIZE_INITIATOR=1
 ROADBLOCK_DEBUG=" --log-level debug "
 ROADBLOCK_IMAGE_NAME=roadblock-client-test
+LEADER_SIGINT_TEST=0
 
-options=$(getopt -o "f:taw" --long "followers:,timeout,abort,wait-for,wait-for-abort,wait-for-heartbeat-timeout" -- "$@")
+options=$(getopt -o "f:taw" --long "followers:,timeout,abort,wait-for,wait-for-abort,wait-for-heartbeat-timeout,leader-sigints:" -- "$@")
```

Comment:
what if the test is running already ?

Reply:
That really isn't applicable here because there are a lot of other existing parts of this that would break if you ran `test.sh` multiple times simultaneously.